monetio/models/__init__.py

```diff
@@ -1,4 +1,5 @@
 from . import (
+    _camx_fv_mm,
```

Comment:
```suggestion
    _camx_mm,
```
I don't "fv" in the file name

Reply:
It should be changed in the `__all__` as well

Reply:
Done